### ISSUE TITLE
Instruments Presets Number not applied in SF2 file

### ISSUE THREAD

**Issue by edrickblade** [git-moss/ConvertWithMoss#36]
Hi, congratulations on your project, I want to report that ConvertWithMoss doesn't recognize the presets number in SF2 files, when convert SF2 files it misses the presets attributes that gives the GM ordering, Ej. if convert 001-Piano it loses the preset number and miss the ordering, 001-Piano is supposed to be the first preset but after converted to any other format it misses the preset number and now is ordered by name and not by GM order number, I hope you can fix this detail because it's hard to find the presets out of GM order.

**Comment by git-moss** [comment]
I could maybe add this as an option.

**Comment by edrickblade** [comment]
> I could maybe add this as an option.

It would be a crucial feature! great!

**Comment by edrickblade** [comment]
Thank for your great work and for this great project1

**Comment by edrickblade** [comment]
Please, can you add the Sf2 preset numbers, please every time we covert or even split a big Sf2  ConvertWithMoss remove the presets numbering making the classification a headache, please, we need it, thanks for your great work!

**Comment by audio2000** [comment]
I was going to make a request post for this as well.
I agree I would like to see this feature as well if would be willing to add this.
Would be great if each patch could get a prefix containing its number.
001-Piano 
might also be nice to have an option with spaces
001 - Piano
or have a field that allows user to be able to enter what kind of separator desired

@edrickblade 
The tricky question is, some soundfonts also contain multiple banks which also have numbers.
Not sure how Convert With Moss handles them, but if the soundfont does contain more than one bank then you might end up with multiples of the same number which could mess up your sort order
001-Piano
001-Violin
I do not think I would want the prefix to include bank number prefix as well, 
001-001-piano
001-002-guitar
001-003-Etc.
002-001-piano
001-002-Etc.
but not sure how else this could be handled?

In the request post I mentioned I was going to make before I saw that this post already existed, I also had another idea that you or other users might find useful.  I will make a reference and link it below if anyone is interested.  It is a solution of what to do if converting multiple soundfonts that each contain multiple patches.  Could be a solution for the issue I just mentioned above as well.

Anyways, yeah this would be great if this was added, this would be very useful.
Currently all the patches get jumbled in the same folder which can get confusing to know where they are from and what order they belong in.

Thank You
